components/grade-table.js

```diff
@@ -1,26 +1,55 @@
 
 class GradeTable  {
-  constructor(tableElement) {
+  constructor(tableElement, noGradesElement) {
    this.tableElement = tableElement;
+   this.noGradesElement = noGradesElement;
   }
 
   updateGrades(grades) {
-    var tbody = this.tableElement.querySelector('tbody');
+    if (grades != 0) {
+      $('p').addClass('d-none');
+    }
+
+    else {
+      $('p').addClass('d-block');
+    }
+
     $(tbody).empty();
 
-    var $header = $('<tr>' + '<td><strong>Name</strong></td>' + '<td><strong>Course</strong></td>' + '<td><strong>Grade</strong></td>' + '</tr>');
+    var $header = $('<tr>' + '<td><strong>Name</strong></td>' + '<td><strong>Course</strong></td>' + '<td><strong>Grade</strong></td>' + '<td><strong>Operations</strong></td>' + '</tr>');
```

Comment:
Your header should be done in a thead so that it doesnt get removed when you empty the body (also better for screen readers!)